app/server/lib/OIDCConfig.ts

```diff
@@ -95,6 +99,11 @@ export class OIDCConfig {
       defaultValue: false,
     })!;
 
+    this._ignoreEmailVerified = section.flag('forceEmailVerified').readBool({
```

Comment:
Could you update the flag's label (`forceEmailVerified`) to `ignoreEmailVerified` too?

Reply:
Sure, done!